definitions/variable/emissions/emissions.yaml

```
@@ -251,18 +266,17 @@
     description: Emissions of {Level-2 Species} from waste (IPCC category 6)
```

```suggestion
    description: Emissions of {Level-2 Species} from waste (IPCC 1996 category 6)
```

A potentially handy cross-reference https://github.com/primap-community/climate_categories/blob/main/climate_categories/data/IPCC1996.yaml. Might be worth pulling up the guidance here that "CO2 from organic waste handling and decay should not be included", i.e. this category is just for emissions from fossil-based sources. (Having said that, if we're not doing this perfectly, let's not spend too much time on it.)

---

Ahh that's very useful to have the IPCC categories in that overview too. 

Should be made clear which guidance is used, because 1996 and 2006 guidance are different. 
I noted this in #186; as we perhaps want to make a separate label for it?

---

Added in d4fad19b34d41528f5db215191d91a999f9566ea